spesifikasjoner/afpant/afpant-model/xslt/restgjeld.xslt

```diff
@@ -0,0 +1,868 @@
+                        </xsl:call-template>
+                    </div>
+                    <div id="body">
+                        <xsl:apply-templates select="innfrielsessaldoforespoersel"/>
+                        <xsl:apply-templates select="innfrielsessaldosvar"/>
+                        <xsl:apply-templates select="restgjeldsforespoersel"/>
+                        <xsl:apply-templates select="restgjeldssvar"/>
+                    </div>
+                </div>
+            </body>
+        </html>
+    </xsl:template>
+    <xsl:template name="overskrift">
+        <xsl:if test="innfrielsessaldoforespoersel">
+            <xsl:text>Forespørsel om innfrielsessaldo</xsl:text>
+        </xsl:if>
+        <xsl:if test="innfrielsessaldosvar">
+            <xsl:text>Svar på innfrielsessaldo</xsl:text>
+        </xsl:if>
+        <xsl:if test="restgjeldsforespoersel">
+            <xsl:text>Forespørsel om restgjeld</xsl:text>
+        </xsl:if>
+        <xsl:if test="restgjeldssvar">
+            <xsl:text>Svar på restgjeld</xsl:text>
+        </xsl:if>
+    </xsl:template>
+    <xsl:template match="/innfrielsessaldoforespoersel">
+        <xsl:call-template name="mottaker"/>
+        <xsl:call-template name="eiendom">
+            <xsl:with-param name="registerenhetsliste" select="registerenheterMedHjemmelshavere/registerenhetMedHjemmelshavere"/>
+        </xsl:call-template>
+        <xsl:call-template name="innfrielsesdatoer"/>
+        <xsl:call-template name="ressurser"/>
+        <xsl:call-template name="avsender"/>
+        <hr/>
+    </xsl:template>
+    <xsl:template match="/innfrielsessaldosvar">
+        <xsl:call-template name="mottaker"/>
+        <xsl:call-template name="eiendom">
+            <xsl:with-param name="registerenhetsliste" select="registerenheterMedDokumentreferanser/registerenhetMedDokumentreferanse"/>
+        </xsl:call-template>
+        <xsl:call-template name="laan"/>
+        <xsl:call-template name="ressurser"/>
+        <xsl:call-template name="avsender"/>
+        <hr/>
+    </xsl:template>
+    <xsl:template match="/restgjeldsforespoersel">
+        <xsl:call-template name="mottaker"/>
+        <xsl:call-template name="eiendom">
+            <xsl:with-param name="registerenhetsliste" select="registerenheterMedHjemmelshavere/registerenhetMedHjemmelshavere"/>
+        </xsl:call-template>
+        <xsl:apply-templates select="prisantydning"/>
+        <xsl:call-template name="ressurser"/>
+        <xsl:call-template name="avsender"/>
+        <hr/>
+    </xsl:template>
+    <xsl:template match="/restgjeldssvar">
+        <xsl:call-template name="mottaker"/>
+        <xsl:call-template name="eiendom">
+            <xsl:with-param name="registerenhetsliste" select="registerenheterMedDokumentreferanser/registerenhetMedDokumentreferanse"/>
+        </xsl:call-template>
+        <xsl:call-template name="laan"/>
+        <xsl:call-template name="ressurser"/>
```

Comment:
Mangler presentasjon av om lånene er sperret for opplån eller ikke?
"sperretForVidereOpplaan"

Reply:
<img width="442" alt="image" src="https://github.com/user-attachments/assets/ab94de8a-f550-40ee-9a9c-d6d1456163c5" />

Mange måter å formattere dette på, men nå vises det iaff

Reply:
"bekreftelsePantrettSlettes" er fjernet fra XSD-en, så det er et ugyldig element.
Sperret for videre opplån tenker jeg er grei å presentere som ja/nei slik du har gjort.